readme.md

```diff
@@ -1 +1,67 @@
+# Gym Landing Page
```

Comment:
Same as [my comment](https://github.com/Min-Ho-Lim/Gym-Landing-Page/pull/28) on #28
Why do you create new readme.md?

Reply:
- [ ] Delete `readme.md` from your changes. We already have `README.md`.

Reply:
I have deleted the readme.md file